src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-subsystems-details/nvmeof-subsystems-details.component.html

```diff
@@ -38,6 +38,20 @@
         </cd-nvmeof-initiators-list>
       </ng-template>
     </ng-container>
+    <ng-container ngbNavItem="performance-details"
+                  *ngIf="permissions.grafana.read">
+      <a ngbNavLink
+         i18n>Performance Details</a>
+      <ng-template ngbNavContent>
+        <cd-grafana i18n-title
+                    title="NvmeOF subsystem details"
```

Comment:
```suggestion
                    title="NvmeoF subsystem details"
```
But I think we can avoid `NvmeoF`  just call `Subsystem Detail` since its under nvme/tcp navigation already.
Similarly at other places.